### PR TITLE
chore: update php max version

### PR DESCRIPTION
**Pull request by qkdreyer**

### CODE REVIEW

**Comment by qkdreyer** [comment]
@Timandes

**Comment by qkdreyer** [comment]
ping

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,11 @@
 language: php
+dist: bionic
 php:
+    - 8.1.0
```
Comment (Timandes) [inline .travis.yml]: Why do you use '8.1.0' instead of '8.0'?

File: .travis.yml [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
     - 8.0
     - 7.3
     - 7.2
     - 7.1
-    - 7.0
```
Comment (Timandes) [inline .travis.yml]: Maybe I've fixed that compilation error in master branch :)
So would you consider save it for a while?

**Review by Timandes — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (qkdreyer) [reply .travis.yml]: https://travis-ci.community/t/php-8-1-support/12439

Comment (qkdreyer) [reply .travis.yml]: 8.1 doesn’t work whereas 8.1.0 does

File: .travis.yml (hunk 2 above)
Comment (qkdreyer) [reply .travis.yml]: Yeah I can revert that if you fixed it !

Comment (qkdreyer) [reply .travis.yml]: Looks like the pho 7.0 build is still failing

**Comment by qkdreyer** [comment]
to sum up :
- php 8.1 is available on travis using `8.1.0` explicitly : https://docs.travis-ci.com/user/languages/php/
- php 8.1 is not working on xenial (16.04) because of libargon2 : https://app.travis-ci.com/github/mustafa-travisci/testBuilds/jobs/551485667#L203
- php 8.1 is working on bionic (18.04), but php 7.0 is not available anymore on that one

**Comment by qkdreyer** [comment]
ping @Timandes

**Comment by zhupeijie** [comment]
您好，邮件已收到，我近期会登录查看，祝您天天开心！

**Comment by qkdreyer** [comment]
Can you also update the 1.0.0 version or release a newer one on PECL please ?

**Comment by zhupeijie** [comment]
您好，邮件已收到，我近期会登录查看，祝您天天开心！

**Pull request merged**